src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs

```diff
@@ -78,7 +78,7 @@ public string CurrentDirectory
 
         protected internal CoreV2.NuGet.IPackageRepositoryFactory RepositoryFactory { get; set; }
 
-        private Lazy<string> MsBuildDirectory {
+        private Lazy<MsBuildToolset> MsBuildDirectory {
```

Comment:
This is no more a msbuild directory.. can you change the name to something more relevant?